backend/app/services/quiz.py

```diff
@@ -0,0 +1,12 @@
+from typing import Sequence
+
+from sqlalchemy import select
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from app.models.quiz import Quiz
+
+
+async def get_quizzes(session: AsyncSession) -> Sequence[Quiz]:
+    statement = select(Quiz).order_by(Quiz.created_at)
+    result = await session.execute(statement)
+    return result.scalars().all()
```

Comment:
There is actually a nice trick to use:
```suggestion
    return await session.scalars(statement).all()
```
(if i'm not mistaken 😄 )

Reply:
`.scalars` is a coroutine so we'd need to do
```python
    return (await session.scalars(statement)).all()
```
which in my opinion is less readable

Reply:
I agree with Radek